src/components/my-reading/article-tracking-card.js

```diff
@@ -167,12 +168,19 @@ export const ArticleTrackingCard = ({
   trackingTitle,
   trackingContent,
   trackingArticleTitle,
+  trackingArticleSlug,
 }) => {
+  const { releaseBranch } = useContext(CoreContext)
+  const handleCardOnClick = () => {
+    const originsForClient = origins.forClientSideRendering[releaseBranch].main
+    window.open(`${originsForClient}/a/${trackingArticleSlug}`, '_blank')
```

Comment:
Could we use entity path defined in `@twreporter/core` package to replace hard-coded `/a` ([ref](https://github.com/twreporter/twreporter-npm-packages/blob/d0b553e3d3df415f0be4bcac782669612828f225/packages/core/src/constants/entity-path.js#L2))

Reply:
fixed